atomic_reactor/utils/koji.py

```diff
@@ -322,7 +324,7 @@ def get_buildroot():
         },
         'container': {
             'type': 'none',
-            'arch': os.uname()[4],
+            'arch': arch or os.uname().machine,
```

Comment:
I think we should just use 'host_arch'

os.uname vs platform.processor give the same results for ppc,x86,s390x, and differ only for arm

arm64 vs aarch64

so I'd choose just one of os.uname or platform.processor as default

Reply:
+1 for choosing one as default, I believe there is also difference between x86_64 vs amd64 representation

Reply:
when I was checking brew medata.json I've seen diff only for aarch64